app/browser/dataset-browse.tsx

```diff
@@ -290,9 +300,23 @@ const NavChip = ({
 
 const encodeFilter = (filter: BrowseFilter) => {
   const { iri, __typename } = filter;
-  return `${
-    __typename === "DataCubeTheme" ? "theme" : "organization"
-  }/${encodeURIComponent(iri)}`;
+  const folder = (() => {
+    switch (__typename) {
+      case "DataCubeTheme":
+        return "theme";
+      case "DataCubeOrganization":
+        return "organization";
+      case "DataCubeAbout":
+        throw new Error("Should not happen");
+      case "Termset":
+        return "termset";
+      default:
+        // eslint-disable-next-line
```

Comment:
Wouldn't an exhaustive check work here without disabling eslint?

Reply:
Yep, it does, it was disabled because previously it was unused, now I use it in the Error. Thanks.